GEOparse/GEOTypes.py

```diff
@@ -726,9 +726,9 @@ def _get_object_as_soft(self):
             soft.append(self.database._get_object_as_soft())
         soft += ["^%s = %s" % (self.geotype, self.name),
                  self._get_metadata_as_string()]
-        for gsm in self.gsms.itervalues():
+        for gsm in self.gsms.values():
             soft.append(gsm._get_object_as_soft())
-        for gpl in self.gpls.itervalues():
+        for gpl in self.gpls.values():
```

Comment:
anyway, this is a very nice project (and actually the first project that I tried to contribute too) :)

Best regards,


Haries.